src/ts/components/quickseekbutton.ts

```diff
@@ -0,0 +1,131 @@
+import { Button, ButtonConfig } from './button';
+import { i18n } from '../localization/i18n';
+import { PlayerAPI } from 'bitmovin-player';
+import { UIInstanceManager } from '../uimanager';
+import { PlayerUtils } from '../playerutils';
+
+export enum QuickSeekDirection {
+  Forward = 'forward',
+  Rewind = 'rewind',
+}
+
+export interface QuickSeekButtonConfig extends ButtonConfig {
+  /**
+   * Specify how many seconds the player should seek forward/backwards in the stream.
+   * Default is 10.
+   */
+  seekSeconds?: number;
+
+  /**
+   * Specify whether the button should fast-forward or rewind.
+   * Default is rewind.
+   */
+  direction?: QuickSeekDirection;
```

Comment:
nit: instead of having the direction separate, we could control everything via the `seekSeconds` config: when `> 0`: direction is forward; when `< 0`: direction is back.

Reply:
Great idea! Didn't think of that 😅 
Done in 345fcb2a